### PR TITLE
Iam changes

### PR DESCRIPTION
**Pull request by sanjeevghimire**
* fixed iam changes for watson assistant
* changed readme
* added new screenshot for invocation name to be consistent with alexa conversation screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +147,20 @@ Edit the `.params` file and add the required settings.
 }
 ```
 
+* If the service credentials from IBM Watson Assistant is IAM based as below, populate the IAM apikey, url, and workspace_id and replace the `.params` file
```
Comment (markstur) [inline README.md]: same thing here re: s/is/are/ and either IAM-based or more clearly say "contain IAM credentials..."

File: README.md [hunk 2]
```diff
@@ -133,6 +133,10 @@ Edit the `.params` file and add the required settings.
 
 #### `params.sample:`
```
Comment (markstur) [inline README.md]: Here's where I think we're going to show `params.sample` and tell them to copy it and edit it.  If not, then there's a bit more to clean up.

File: README.md (hunk 1 above)
Comment (sanjeevghimire) [reply README.md]: ```If the service credentials from IBM Watson Assistant are IAM based as shown in the diagram below, populate the IAM apikey, url, and workspace_id fields  and replace the `.params` file with the populated JSON ``` how about this @markstur ?

File: README.md [hunk 3]
```diff
@@ -133,6 +133,10 @@ Edit the `.params` file and add the required settings.
...
 
+* If the service credentials from IBM Watson Assistant is username/password based as below populate the username, password and workspace_id and replace `.params` file with the json
```
Comment (markstur) [inline README.md]: Here's where to be nit-picky "credentials" is plural so "are" not "is". Also "based" would be a hyphen suffix thingy when used like this so username/password-based... but that is a little ugly so I suggested just say something simple like "if the service credentials from Wassistant contain username/password, then populate u,p and w_id in the .params file.

**Pull request merged**